ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/radius-server.j2

```diff
@@ -0,0 +1,46 @@
+{# eos - radius servers #}
+{# eos - radius servers #}
+{% if radius_server is arista.avd.defined %}
+!
+{%     if radius_server.attribute_32_include_in_access_req is arista.avd.defined %}
+{%         set attribute_32_include_in_access_cli = "radius-server attribute 32 include-in-access-req" %}
+{%         if radius_server.attribute_32_include_in_access_req.hostname is arista.avd.defined (true) %}
+{%             set attribute_32_include_in_access_cli = attribute_32_include_in_access_cli ~ " hostname" %}
+{%         endif %}
+{%         if radius_server.attribute_32_include_in_access_req.format is arista.avd.defined and not radius_server.attribute_32_include_in_access_req.hostname is arista.avd.defined %}
+{%             set attribute_32_include_in_access_cli = attribute_32_include_in_access_cli ~ " format " ~ radius_server.attribute_32_include_in_access.format %}
+{%         endif %}
+{{ attribute_32_include_in_access_cli }}
+{%     endif %}
+{%     if radius_server.dynamic_authorization is arista.avd.defined %}
+{%         set dynamic_authorization_cli = "radius-server dynamic-authorization" %}
+{%         if radius_server.dynamic_authorization.port is arista.avd.defined %}
+{%             set attribute_32_include_in_access_port_cli = dynamic_authorization_cli ~ " port " ~ radius_server.dynamic_authorization.port %}
+{{ attribute_32_include_in_access_port_cli }}
+{%         endif %}
+{%         if radius_server.dynamic_authorization.tls_ssl_profile is arista.avd.defined %}
+{%             set dynamic_authorization_cli = dynamic_authorization_cli ~ " tls ssl-profile " ~ radius_server.dynamic_authorization.tls_ssl_profile %}
+{{ dynamic_authorization_cli }}
+{%         endif %}
+{%     endif %}
+{%     if radius_server.hosts is arista.avd.defined %}
+{%         for radius_servers in radius_server.hosts %}
+{%             if radius_servers.host is arista.avd.defined %}
+{%                 set radius_cli = "radius-server host " ~ radius_servers.host %}
+{%             endif %}
+{%             if radius_servers.vrf is arista.avd.defined %}
+{%                 set radius_cli = radius_cli ~ " vrf " ~ radius_servers.vrf %}
+{%             endif %}
+{%             if radius_servers.timeout is arista.avd.defined %}
+{%                 set radius_cli = radius_cli ~ " timeout " ~ radius_servers.timeout %}
+{%             endif %}
+{%             if radius_servers.retransmit is arista.avd.defined %}
+{%                 set radius_cli = radius_cli ~ " retransmit " ~ radius_servers.retransmit %}
+{%             endif %}
+{%             if radius_servers.key is arista.avd.defined %}
+{%                 set radius_cli = radius_cli ~ " key 7 " ~ radius_servers.key %}
+{%             endif %}
+{{ radius_cli }}
+{%         endfor %}
+{%     endif %}
```

Comment:
Our `natural_sort` filter both sorts on key and check if the list is even defined. It will return an empty list if not.
The `host` key should be required. Make sure to set it as `primary_key` in the schema for the `radius_server.hosts` list.
To avoid confusing naming, I am renaming the vars below.

```suggestion
{%     for radius_host in radius_server.hosts | arista.avd.natural_sort('host' %}
{%         set radius_cli = "radius-server host " ~ radius_host.host %}
{%         if radius_host.vrf is arista.avd.defined %}
{%             set radius_cli = radius_cli ~ " vrf " ~ radius_host.vrf %}
{%         endif %}
{%         if radius_host.timeout is arista.avd.defined %}
{%             set radius_cli = radius_cli ~ " timeout " ~ radius_host.timeout %}
{%         endif %}
{%         if radius_host.retransmit is arista.avd.defined %}
{%             set radius_cli = radius_cli ~ " retransmit " ~ radius_host.retransmit %}
{%         endif %}
{%         if radius_host.key is arista.avd.defined %}
{%             set radius_cli = radius_cli ~ " key 7 " ~ radius_host.key %}
{%         endif %}
{{ radius_cli }}
{%     endfor %}
```